analyses/fusion_filtering/run_fusion_merged.sh

```diff
@@ -99,8 +99,10 @@ Rscript 02-fusion-filtering.R --standardFusionFiles $standard_starfusion_file,$s
 # Fusion zscore annotation for filtered fusion for the combined RNA expression file
 Rscript 03-Calc-zscore-annotate.R --standardFusionCalls "${scratch_path}/standardFusionExp_QC_expression_filtered_annotated.RDS" \
                                   --expressionMatrix $rna_expression_file \
-                                  --normalExpressionMatrix $normal_expression_file \
-                                  --outputFile "${scratch_path}/standardFusionExp_QC_expression"
+                                  --clinicalFile $histologies_file \
+                                  --cohortInterest "PBTA,GMKF" \
+                                  --normalExpressionMatrix $normal_expression_brain,$normal_expression_adrenal_gland \
+                                  --outputfile "${scratch_path}/standardFusionExp_QC_expression"
```

Comment:
this should be `--outputFile` I think from the help section

Reply:
Yes you are correct! Fixed now!